lib/src/git.rs

```diff
@@ -102,6 +103,22 @@ impl GitSettings {
     }
 }
 
+/// Configuration for a Git subprocess
+#[derive(Debug)]
```

Comment:
nit: also add `derive(Clone)`? It's unlikely this type will be uncloneable.

Reply:
added